migrator.go

```diff
@@ -108,7 +96,18 @@ func (m spannerMigrator) CreateTable(values ...interface{}) error {
 				values                  = []interface{}{m.CurrentTable(stmt)}
 				hasPrimaryKeyInDataType bool
 			)
-
+			for _, f := range stmt.Schema.Fields {
+				// Cloud spanner does not support auto incrementing primary keys.
+				if f.AutoIncrement && f.HasDefaultValue && f.DefaultValue == "" && f.DefaultValueInterface == nil {
+					if err := tx.Exec("CREATE SEQUENCE IF NOT EXISTS " +
+						stmt.Table + "_GORM_SEQUENCE" +
```

Comment:
What is the max table name length and max sequence name length? Put another way: would it make sense to make the sequence name as 'short' as possible, by using something like `TableName + "_seq"`, in order to reduce the probability that a long table name will cause the sequence creation to fail because the name is too long?

Also; Should we add support for an annotation to set a custom sequence name? That won't work with `gorm.Model`, but would work with anyone who creates their own model and wants to use a custom sequence name.